### DISCUSSION TITLE
Kubernetes Seq Kong Ingress definition

### DISCUSSION THREAD

**Discussion by whebz** [datalust/seq-tickets#1521] [Help]
Hello,

I was trying to figure out how to configure ingress definition in my container seq running on AKS with kong ingress.
I can't make it work. Can someone provide me an example?

thanks

config.yaml: 

persistence:
  storageClass: managed-premium
  size: 20Gi
gelf:
  enabled: true
service:
  type: ClusterIP
  ingress:
   annotations:
     [kubernetes.io/ingress.class](http://kubernetes.io/ingress.class): kong
  ui:
    ingress:
      enabled: true
      path: /app/seq/
      hosts:
        - https://domainname
  ingestion:
    ingress:
      enabled: true
      path: /app/seq/
      hosts:
        - https://domainname

**Comment by KodrAus** [comment]
Hi @whebz :wave: I'm not familiar with kong myself, but we have a Helm chart [here](https://github.com/datalust/helm.datalust.co) along with a sample that uses NGINX as an ingress controller [here](https://github.com/datalust/helm.datalust.co/tree/main/samples). That might give you a starting point for your own deployment.


Hope that helps 🙂
